docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc

```diff
@@ -31,28 +31,28 @@ $ sudo cephadm shell -- ceph mgr fail <mgr_instance>
 +
 ----
 $ for label in mon mgr _admin; do
-    sudo cephadm shell -- ceph orch host rm label <source_node> $label;
+    sudo cephadm shell -- ceph orch host label rm <source_node> $label;
 done
 ----
 +
 * Replace `<source_node>` with the hostname of the source node.
 
-. Remove the running Ceph Monitor daemon from the source node:
+. (Optional) Ensure that you remove the Ceph Monitor daemon from the source node if it is still running:
```

Comment:
```suggestion
. Optional: Ensure that you remove the Ceph Monitor daemon from the source node if it is still running:
```
Our style for optional steps is "Optional:", not "(Optional)."

Reply:
done